docs/build_from_source.md

```diff
@@ -113,37 +113,18 @@ Example:
 ```bash
 make docker_build NVIDIA=1 OV_USE_BINARY=0 OV_SOURCE_BRANCH=releases/2022/3 OV_CONTRIB_BRANCH=releases/2022/3
 ```
-```bash
-docker run -it --gpus all -p 9178:9178 -v ${PWD}/models/public/resnet-50-tf:/opt/model openvino/model_server:latest-cuda --model_path /opt/model --model_name resnet --target_device NVIDIA
-
-<hr />
-
-### `SENTENCEPIECE`
```

Comment:
Why do we remove SENTENCEPIECE from documentation? The parameter will become undocumented and default value is 0 so we leave users with no information how to get extension for MUSE.